neo/api/JSONRPC/JsonRpcApi.py

```diff
@@ -106,5 +106,10 @@ def get_error_payload(self, request_id, error, data=None, code=None):
     def json_rpc_method_handler(self, method, params):
         # print("method", method, params)
         if method == "getblockcount":
-            return str(Blockchain.Default().HeaderHeight)
+            return Blockchain.Default().HeaderHeight
+        if method == "getblockhash":
+            height = params[0]
+            if height >= 0 and height <= Blockchain.Default().Height:
+                return Blockchain.Default().GetBlockHash(height).decode('utf-8')
```

Comment:
Please add an `else` block here, and throw a meaningful error ("block number too high" or smth). Like this it would return the "Method not found" error